modules/es-extensions/publisher/asset/wsdl/asset.js

```diff
@@ -255,8 +255,29 @@ asset.manager = function(ctx) {
             
         },
         getVersion: function(asset) {
+            if (!asset.attributes["version"]) {
+                var subPaths = asset.path.split('/');
+                asset.version = subPaths[subPaths.length - 2];
+                asset.attributes["version"] = asset.version;
```

Comment:
Thanks for pointing it out. Fixed in the following commit - https://github.com/wso2/product-greg/pull/851/commits/63fa2884c0274e0aea58d2866cc0a55d4b53e55c